src/frontend/src/components/pinInput.ts

```diff
@@ -112,15 +116,21 @@ export const pinInput = <T>({
       >
         ${Array.from(
           { length: pinLength },
+          /* XXX: we use a special class/font for the 'secret' PIN input instead of setting
+           * type="password", otherwise the browser tries to save one char as a password */
           (_, _ix) => html`
             <li class="c-list--pin-char c-input--anchor">
               <label class="c-input--anchor__wrap">
                 <input
+                  ?autofocus=${focus}
                   autocomplete="off"
```

Comment:
should `autocapitalize` and `autocorrect` be turned `off` is the input becomes `text`?

Reply:
good catch (modulo `autocorrect`/`spellcheck`)